airbrakes/data_handling/data_processor.py

```diff
@@ -132,13 +144,18 @@ def update(self, data_points: list[EstimatedDataPacket]) -> None:
                 ]
             )
             # TODO: Comment this line out before flight!
```

Comment:
Do you think you could add the gravity vectors to the csv files and remove this?